src/Illuminate/Foundation/helpers.php

```diff
@@ -323,7 +323,7 @@ function cookie($name = null, $value = null, $minutes = 0, $path = null, $domain
      */
     function csrf_field()
     {
-        return new HtmlString('<input type="hidden" name="_token" value="'.csrf_token().'" autocomplete="off">');
+        return new HtmlString('<input type="text" style="display:none" name="_token" value="'.csrf_token().'">');
```

Comment:
I would argue that simply removing the autocomplete attribute would be best. Hidden input types are an established type and changing this could potentially have reaching effects.

Reply:
See #48371

Reply:
Please, respect W3C!